spec/shoryuken/worker_spec.rb

```diff
@@ -99,36 +100,64 @@
   end
 
   describe '.shoryuken_options' do
-    it 'registers a worker' do
-      expect(Shoryuken.worker_registry.workers('default')).to eq([TestWorker])
+    let(:dummy_worker) { Class.new }
+
+    before do
+      dummy_worker.include Shoryuken::Worker
     end
 
-    it 'accepts a block as queue name' do
-      $queue_prefix = 'production'
+    subject { dummy_worker.shoryuken_options shoryuken_options }
+
+    context "when using the queue key" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.